scripts/lib/CIME/utils.py

```diff
@@ -225,8 +225,7 @@ def _convert_to_fd(filearg, from_dir):
 
 _hack=object()
 
-def run_sub_or_cmd(cmd, cmdargs, subname, subargs, logfile=None, case=None,
-                   from_dir=None, combine_output=False):
+def run_sub_or_cmd(cmd, cmdargs, subname, subargs, logfile=None, case=None, from_dir=None):
```

Comment:
Should we leave the combine_output flag in place and mark it deprecated so that we don't break any external users - I don't actually know that there are any but...

Reply:
I think we can just remove it. I thought external users used CIME through it's script interface?